packages/cli/lib/projects.js

```diff
@@ -168,14 +168,51 @@ const validateProjectConfig = (projectConfig, projectDir) => {
   }
 };
 
+const pollFetchProject = async (accountId, projectName) => {
```

Comment:
Going off of the screenshot here, it was my understanding that the issue was that we are not able to successfully create the project in the newly-created sandboxes. This seems to be adding polling to the fetching of the project, not the creation of it in the target account. Let me know if I'm just misunderstanding what the original issue was!

![Screenshot 2023-05-08 at 15 53 30](https://github.com/HubSpot/hubspot-cli/assets/6654014/db422cd8-c2a9-478c-9e66-1b37e7bc0a19)

Reply:
Yep its both! The very last prompt was asking me to create a project, but then it fails and exits. This is before you merged in your other branch with an error message though. 

By adding the polling to the fetch, we can ensure that upon a 404 or success, the portal has been successfully ungated and project create in the next step will work

Reply:
Yeah if its not ungated, we'll get a 403 with the following message: 
![Screenshot 2023-05-08 at 15 28 17](https://github.com/HubSpot/hubspot-cli/assets/16788677/c7848b6d-e668-4684-9a18-73bf6e71b743)